src/signal.js

```diff
@@ -1,3 +1,16 @@
+/**
+ * Signals
```

Comment:
Glorious documentation, reads very nice!

Reply:
Glad you like it!